README.md

```diff
@@ -1,23 +1,118 @@
 # Kuri EDU
-A minimal kuri robot that exposes kuri's subsystems without any autonomous behavior
+This repository contains software and activities intended to be used for
+educational purposes.  The kuri_edu folder contains a ROS package that provides 
+a minimal running robot framework and the activities folder contains ideas and
+instructions for extending the robot behavior
 
-## Launch Files
- * kuri_remote.launch: Launches a Kuri that can be driven with the standard keyboard_teleop node
- * kuri_edu.launch: Launches kuri_remote along with the following subsystems
-   * madmux: Kuri's camera
+## Stand-alone launch Files
+ * kuri_edu.launch: Launches Kuri, and the nodes required to operate Kuri with 
+minimal autonomous behavior.  This launch file glues together several other launch files:
+    * kuri_drive.launch - Runs the hardware abstraction layer and depth-sensor.
+    * madmux_daemon.launch - Provides a hardware abstraction layer to Kuri's camera
+    * vision_bridge.launch - Performs face and object detection
+
+## Getting Started:
+Before attempting any activities, you should make sure you're able to launch the
+kuri EDU software without any modifications.
+
+Start by establishing an SSH connection to Kuri
+`ssh mayfield@kuri-SERIAL_NUMBER`
+
+To set up your shell, source the setup.sh or setup.bash files in the kuri EDU
+package
+`source /opt/edu/setup.sh`
+
+Finally, launch the kuri_edu software
+`roslaunch kuri_edu kuri_edu.launch`
+
+This should launch a ROS master, and some ROS nodes:
+```
+mayfield@kuri-0001c:~$ roslaunch kuri_edu kuri_edu.launch &
+[1] 19589
+mayfield@kuri-0001c:~$ ... logging to /home/mayfield/.ros/log/8997b7cc-bde9-11e8-9393-00073249657e/roslaunch-kuri-0001c-19589.log
+Checking log directory for disk usage. This may take awhile.
+Press Ctrl-C to interrupt
+Done checking log file disk usage. Usage is <1GB.
+
+started roslaunch server http://kuri-0001c:36850/
+
+. . .
+
+process[robot_state_publisher-2]: started with pid [19637]
+process[mobile_base-3]: started with pid [19638]
+process[depthscan-4]: started with pid [19639]
+process[madmux_daemon-5]: started with pid [19645]
+process[vision_bridge-6]: started with pid [19658]
+process[safety_controller-7]: started with pid [19665]
+process[head_controller-8]: started with pid [19668]
+process[chest_light_controller-9]: started with pid [19678]
+```
+
+At this point, you should observe that your Kuri robot's eyes open, the head 
+looks up, and the chest light begins to play a simple pattern.  The color of 
+the chest LED represents Kuri's battery level.  Green = 100%, Red = 20% or 
+lower.  Kuri should also back up when the bump shell is pressed.
+
+You can stop the kuri_edu software with ctrl-c
 
 ## Keyboard Teleop Demo
 
-To keyboard teleop a Kuri, run kuri_remote.launch or kuri_edu.launch on the robot:
+You can also drive Kuri like a remote control car using the 
+teleop_twist_keyboard node.  You may need to install this node on Kuri.
```

Comment:
They won't have the ROS `apt` sources at this point. You may wanna point to ROS Indigo wiki or our docs. Oh, you've mentioned this below. Maybe say "see below if you are having trouble installing this" here.

Reply:
Done on line 75